nubis/builder/provisioners.json

```diff
@@ -2,26 +2,29 @@
 "provisioners": [
   {
     "type": "shell",
-    "inline": ["sudo chmod 777 /var/www/dpaste"],
-    "order": "20"
+    "inline": [
+      "sudo mkdir -p /var/www/dpaste",
```

Comment:
This directory is created by the apache puppet module. This chmod 777 thing is  a hack as the install.sh script runs as the ubuntu user and we need write permissions when copying the dpaste files over. This can be fixed by setting the docroot_owner and  docroot_group  as overrides to the vhost definition. Also if you wish to keep the mode 777 you can set that in the vhost define as well using  docroot_mode.

https://forge.puppetlabs.com/puppetlabs/apache#defined-type-apachevhost